### PR TITLE
Add support for Remix v2

### PR DESCRIPTION
**Pull request by roger-tbg**
changed package-json version to 3.6.0
updated remix* packages to 2.x and 18.x

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "remix-auth",
-  "version": "3.5.1",
+  "version": "3.6.0",
```
Comment (sergiodxa) [inline package.json:3]: This is changed automatically with a GitHub Action
```suggestion
  "version": "3.5.1",
```

**Review by sergiodxa — APPROVED** [review]
Thanks for this, just one comment to revert the version on package.json, a GitHub Action bump that and create a Git tag for the version.

File: package.json (hunk 1 above)
Comment (roger-tbg) [reply package.json]: Done!  Thanks for the feedback

File: package.json [hunk 2]
```diff
@@ -37,20 +37,21 @@
     "README.md"
   ],
   "peerDependencies": {
-    "@remix-run/react": "^1.0.0",
-    "@remix-run/server-runtime": "^1.0.0"
+    "@remix-run/react": "^2.0.1",
+    "@remix-run/server-runtime": "^2.0.1"
```
[the remark this answers is not in the document]
Comment (roger-tbg) [reply package.json]: Done / Thanks for the feedback!

**Comment by Soleone** [comment]
I'm on the edge of my seat watching 3.6.0 make it into production so I can `npm install remix-auth` in my Remix2 project 🤞

**Comment by Soleone** [comment]
Fixes https://github.com/sergiodxa/remix-auth/issues/242

**Comment by wolthers** [comment]
Any idea when this will be released? :)

**Comment by sergiodxa** [comment]
When I have time to fix [CI not working here](https://github.com/sergiodxa/remix-auth/actions/runs/6318473267), or someone makes a PR.

**Comment by aydrian** [comment]
> When I have time to fix [CI not working here](https://github.com/sergiodxa/remix-auth/actions/runs/6318473267), or someone makes a PR.

Hey @sergiodxa, I'm almost finished with a PR to resolve the CI issues. I'll push it either today or tomorrow. I'm currently only updating the dependencies required to fix the issues. Would you like me to also bump the others? The way things are working right now, we are only testing against the 2.x version of Remix. I can follow up with 1.x testing using environments afterward if you think it's worth it.

**Pull request merged**